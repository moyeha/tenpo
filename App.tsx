import { StatusBar } from "expo-status-bar";
import { StyleSheet, Text, View } from "react-native";
import { SafeAreaProvider } from "react-native-safe-area-context";
import { useFonts } from "expo-font";
import AnimatedStyleUpdateExample from "./Animate";

export default function App() {
  const [fontsLoaded] = useFonts({
    "Roboto-Black": require("./assets/fonts/Roboto-Bold.ttf"),
    "Gotham-Black": require("./assets/fonts/Gotham-Bold.otf"),
    "Gotham-Light": require("./assets/fonts/Gotham-Light.otf"),
  });

  return (
    <SafeAreaProvider>
      {/* <View style={styles.container}>
        <Text style={{ fontFamily: "Gotham-Black"}}>Open up App.tsx to start working on your app!</Text>
        <StatusBar style="auto" />
      </View> */}
      {/* <AnimatedStyleUpdateExample/> */}
    </SafeAreaProvider>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: "#fff",
    alignItems: "center",
    justifyContent: "center",
  },
});
