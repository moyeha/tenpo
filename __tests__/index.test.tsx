import React from "react";
import renderer from "react-test-renderer";

// jest.mock("expo-router");

// jest.mock("react-i18next", () => ({
//   // this mock makes sure any components using the translate hook can use it without a warning being shown
//   useTranslation: () => {
//     return {
//       t: (str) => str,
//       i18n: {
//         changeLanguage: () => new Promise(() => {}),
//       },
//     };
//   },
//   initReactI18next: {
//     type: "3rdParty",
//     init: () => {},
//   },
// }));

import App from "../app/index";

describe("<App />", () => {
  it("has 1 child", () => {
    const tree = renderer.create(<App />).toJSON();

    expect(tree.children.length).toBe(1);
  });
});
