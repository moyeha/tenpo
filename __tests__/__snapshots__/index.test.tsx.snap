// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Deberia estar el titulo Tenpo 1`] = `null`;
